Common subdirectories: orig/etc and new/etc
diff -u orig/gc-ctrl.py new/gc-ctrl.py
--- orig/gc-ctrl.py	2017-07-19 21:28:30.067585593 +0000
+++ new/gc-ctrl.py	2017-07-19 21:29:31.583649685 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 # 
 # Copyright 2010-2011 University of Chicago
 # 
diff -u orig/gc.py new/gc.py
--- orig/gc.py	2017-07-19 21:28:30.067585593 +0000
+++ new/gc.py	2017-07-19 21:30:24.191704590 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python -u
+#!/usr/bin/env python
 
 #  Copyright 2010-2011 University of Chicago
 #
Common subdirectories: orig/gt_amd64 and new/gt_amd64
Common subdirectories: orig/gt_i386 and new/gt_i386
diff -u orig/setup.py new/setup.py
--- orig/setup.py	2017-07-19 21:28:30.063585588 +0000
+++ new/setup.py	2017-07-19 21:30:43.507724772 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python -u
+#!/usr/bin/env python
 
 
 #
Common subdirectories: orig/tablelist5.9 and new/tablelist5.9
Common subdirectories: orig/util and new/util